src/Tests/Mission/joined_mission.test.js

```diff
@@ -0,0 +1,16 @@
+import TestRenderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import { Provider } from 'react-redux';
+import store from '../../redux/configureStore';
```

Comment:
* Kindly fix these failing tests I get when I run `npm run test` 
![image](https://user-images.githubusercontent.com/52497246/215957428-7ddc960e-5908-4360-9874-4ac4bcd48aa3.png)

Reply:
Thankyou @Davidosky007 We have fixed this problem.